docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,357 @@
+
+* If you've got a task with ``FAILED`` status and the logs in the worker are of this form:
+
+  .. code-block:: py3
+
+   substrapp.exceptions.PodReadinessTimeoutError: Pod substra.ai/pod-name=substra-***-compute-*** failed to reach the \"Running\" phase after 300 seconds."
+
+  Your Docker disk image might be full, increase it or clean it with ``docker system prune -a``
+
+Troubleshooting deployment
+--------------------------
+
+Skaffold version 1.31.0
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Status check is broken in version 1.31.0 and kubectl secret manifests are not applied until helm deploy is done, but helm deploy depends on kubectl secret manifests.
+It has been fixed in `Skaffold 1.32.0 (PR #6574) <https://github.com/GoogleContainerTools/skaffold/releases/tag/v1.32.0>`__.
+
+The solution for the version 1.31.0 is to add ``--status-check=false`` when running Skaffold:
+
+.. code-block:: bash
+
+   skaffold dev/run/deploy --status-check=false
+
+Failed calling webhook ``validate.nginx.ingress.kubernetes.io``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you encounter the following error message when deploying the backend(s):
+
+
+.. code-block:: bash
+
+   Error: UPGRADE FAILED: failed to create resource: Internal error occurred: failed calling webhook "validate.nginx.ingress.kubernetes.io": an error on the server ("") has prevented the request from succeeding
+   failed to deploy: install: exit status 1
+
+As a workaround, you can delete the failing webhook by launching the following command:
+
+.. code-block:: bash
+
+   kubectl delete Validatingwebhookconfigurations ingress-nginx-admission
+
+You should now be able to :ref:`deploy the backend(s) again<Deploy the backend>`.
+
+Other errors during backend deployment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you encounter one of the following errors while deploying the backend:
+
+.. code-block:: bash
+
+   Error: UPGRADE FAILED: cannot patch "orchestrator-org-1-server" with kind Certificate: Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp <ip>:443: connect: connection refused
+   deploying "orchestrator-org-1": install: exit status 1
+
+.. code-block:: bash
+
+   Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": x509: certificate signed by unknown authority
+
+Check that the orchestrator is deployed and relaunch the command ``skaffold run``.
+
+Troubleshooting monitoring
+--------------------------
+
+k9s limits on log lines
+^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, k9s limits the log to the last 200 lines. To increase this value, set ``logger.tail`` and ``logger.buffer`` to the desired number (e.g. 5000) in the `k9s config file <https://github.com/derailed/k9s#k9s-configuration>`_.
```

Comment:
This is really some advanced usage of the tools that can be used and I'm not sure it should be part of our documentation.